geomstats/geometry/discrete_curves.py

```diff
@@ -70,6 +75,23 @@ def each_belongs(pt):
 
         return each_belongs(point)
 
+    def projection(self, point):
+        """Project a point in ambient manifold on manifold.
+
+        This method is for compatibility and returns the input point unchanged.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., dim_embedding]
+            Point in embedding manifold.
```

Comment:
Maybe we should avoid using "embedding manifold" when nothing is inheriting from EmbeddedManifold?